media-sound/supercollider/supercollider-3.8.0.ebuild

```diff
@@ -23,25 +23,37 @@ RDEPEND="
 	media-libs/alsa-lib
 	sys-libs/readline:0=
 	x11-libs/libXt
+	x11-libs/libX11
 	avahi? ( net-dns/avahi )
 	fftw? ( sci-libs/fftw:3.0= )
-	jack? ( media-sound/jack-audio-connection-kit )
+	jack? ( virtual/jack )
 	portaudio? ( media-libs/portaudio )
 	qt5? (
 		dev-qt/qtcore:5
 		dev-qt/qtgui:5
-		dev-qt/qtpositioning:5
-		dev-qt/qtsensors:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtprintsupport:5
 		dev-qt/qtwebkit:5
+		dev-qt/qtwidgets:5
 	)
 	sndfile? ( media-libs/libsndfile )
-	wiimote? ( app-misc/cwiid )"
+	wiimote? ( app-misc/cwiid )
+"
 DEPEND="${RDEPEND}
 	dev-libs/icu
 	virtual/pkgconfig
 	emacs? ( virtual/emacs )
 	gedit? ( app-editors/gedit )
-	vim? ( app-editors/vim )"
+	vim? ( app-editors/vim )
+	qt5? (
+		dev-qt/linguist-tools:5
+		dev-qt/qtconcurrent:5
+		dev-qt/qtdeclarative:5
+		dev-qt/qtpositioning:5
+		dev-qt/qtopengl:5
+		dev-qt/qtsensors:5
```

Comment:
Hard to tell, but I checked the binaries. after a build (using the qt5 useflage) and these dependencies couldn't be found, but they where missing when I wanted to build the whole thing.
I can check again if you want.